components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/java/org/wso2/carbon/identity/mgt/endpoint/client/ApiClient.java

```diff
@@ -444,6 +448,10 @@ private ClientResponse getAPIResponse(String path, String method, List<Pair> que
             throw new ApiException(500, "Cannot have body and form params");
         }
 
+        if(!isRequestMethodSupported(method, GET, POST, PUT, DELETE, PATCH)) {
```

Comment:
Formatting issue.

Reply:
GET, POST, PUT, DELETE, PATCH needs not to be passed to the method. Instead, `isRequestMethodSupported` method itself needs to decide.

Reply:
Fixed in eba65da2552638e44d143ecbc49837893f931011.